src/templates/app/cashier/paymentagent_withdraw.jsx

```diff
@@ -23,12 +27,50 @@ const PaymentAgentWithdraw = () => (
             </div>
 
             <div id='viewSuccess' className='viewItem invisible'>
-                <p id='successMessage' />
-                <p>
-                    <a className='button' href={it.url_for('user/statementws')}>
-                        <span className='button'>{it.L('View your statement')}</span>
-                    </a>
-                </p>
+                <Fieldset>
+                    <div className='gr-row'>
+                        <div className='gr-8 gr-12-m gr-centered gr-padding-30 gr-child center-text'>
+                            <h2>{it.L('Your request is being processed.')}</h2>
+                            <p id='successMessage' />
+                        </div>
+                    </div>
+
+                    <div className='gr-row'>
+                        <div className='gr-11 gr-centered'>
+                            <SeparatorLine show_mobile />
+                        </div>
+                    </div>
+
+                    <div className='gr-row'>
+                        <div className='gr-8 gr-12-m gr-centered gr-padding-10 gr-child center-text'>
+                            <p>{it.L('Please contact your payment agent to validate your withdrawal request.')}</p>
+                        </div>
+                    </div>
+
+                    <div className='gr-row'>
+                        <div className='gr-push-3 gr-push-0-m gr-7 gr-12-m gr-padding-10'>
+                            <div className='gr-row'>
+                                <div className='gr-2 gr-3-m'>
+                                    <img className='responsive' src={it.url_for('images/pages/contact/contact-icon.svg')} />
+                                </div>
+                                <div className='gr-10 gr-9-m'>
+                                    <h5 id='agentName' />
+                                    <h5 className='gr-padding-20 gr-child' id='agentWebsite'>
+                                        {it.L('Website: ')}<a />
+                                    </h5>
+                                    <h5 className='gr-padding-20 gr-child' id='agentEmail'>
+                                        {it.L('Email: ')}<a />
+                                    </h5>
+                                    <h5 className='gr-padding-20 gr-child' id='agentTelephone'>
+                                        {it.L('Tel: ')}<a />
```

Comment:
why is there extra space after `:`? 😅

Reply:
Extra space after `:` because that's how how its formatted, though I realize it might be better to put it after the translation tag